cpp/zotaut/journal_timeliness_checker.cc

```diff
@@ -47,23 +49,69 @@ bool MaxDeliveredAtSmallerThanUpdateWindow(const time_t &max_delivered_at, const
 }
 
 
+
+std::unique_ptr<KeyValueDB> CreateOrOpenKeyValueDB(const std::string &name) {
```

Comment:
This function looks very generic, it could be moved to KeyValueDb.h and .cc as a static factory method.